src/Components/Navbar/Navbar.css

```diff
@@ -0,0 +1,28 @@
+@import url(/src/index.css);
```

Comment:
There's no need to import `index.css` here if you have already imported it in `index.js`